app/views/expenses/_expense.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= link_to category_expense_path(category_id: params[:category_id], id: expense.id) do %>
+<div id="<%= dom_id expense %>", class="w-full bg-white px-5 py-3 flex flex-col mb-5 hover:bg-[#3778c2] hover:text-white">
+<div class="flex justify-between text-lg">
+  <p>
+    <%= expense.name %>
+  </p>
+
+  <p>
+    $<%= expense.amount %>
+  </p>
+</div>
+    <p class="text-sm">
```

Comment:
- [ ] In this page, Please show the date in UI that when this particular transaction happens as per the design.

### Your design
![Screenshot from 2022-04-30 00-34-50](https://user-images.githubusercontent.com/26767383/166027051-ba196574-8e14-4f3e-89fa-addc55d7e5ab.png)



### GIven design
![Screenshot from 2022-04-30 00-16-20](https://user-images.githubusercontent.com/22434660/166008352-3516ca31-ee95-498f-b454-cb0b444830c2.png)